repair/repair_task.hh

```diff
@@ -57,18 +57,16 @@ private:
     dht::token_range_vector _ranges;
     std::unordered_map<dht::token_range, repair_neighbors> _neighbors;
```

Comment:
This commit should probably be merged with the last one ([ecf80412](https://github.com/scylladb/scylladb/pull/12085/commits/ecf804125fa034f31cc6ed9de9918522cc74e208)).